docs/source/recipes/plot_19_recipe.py

```diff
@@ -0,0 +1,111 @@
+"""
+Plotting per-season trends in global sea surface tempreature extrema
+====================================================================
+
+In this recipe we find the area-based extrema of global sea surface
+temperature per month and, because it is very difficult to
+interpret for trends when in a monthly form, we calculate and plot
+on top of this the mean across each season for both the minima and the
+maxima.
+"""
+
+# %%
+# 1. Import cf-python, cf-plot and other required packages:
+import cfplot as cfp
+import matplotlib.pyplot as plt
+
+import cf
+
+# %%
+# 2. Read the dataset in extract the SST Field from the FieldList:
+f = cf.read("~/recipes_break/ERA5_monthly_averaged_SST.nc")
```

Comment:
Forgot to update this, let me see if I can update it via a suggestion to myself to avoid branch switching etc. :smile: :

```suggestion
f = cf.read("~/recipes/ERA5_monthly_averaged_SST.nc")
```